### PR TITLE
updated syntax

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Summary

Resolves issue #16662 
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by gullapalli9482** [comment]
I think I made pr in a different branch. Thanks for helping me out @wbamberg :). I'll make the next pr containing more than one file unlike this one, this was one just to make sure I was following the process properly.

File: files/en-us/web/api/client/url/index.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ self.addEventListener('notificationclick', function(event) {
   event.waitUntil(clients.matchAll({
     type: 'window'
   }).then(function(clientList) {
-    for (var i = 0; i < clientList.length; i++) {
-      var client = clientList[i];
+    for (let i = 0; i < clientList.length; i++) {
+      let client = clientList[i];
```
Comment (wbamberg) [inline files/en-us/web/api/client/url/index.md:37]:
```suggestion
      const client = clientList[i];
```

**Review by wbamberg — COMMENTED** [review]
> I think I made pr in a different branch this time.

Yes, this looks better :).

> Thanks for helping me out @wbamberg :).

You're welcome. It helps to get into the habit of that pattern I suggested.

Just one `let` here that could  be a `const` I think (it's not being reassigned).

**Review by wbamberg — APPROVED** [review]
👍 looks great, thank you @gullapalli9482 !

**Pull request merged**